src/content/docs/en/reference/cli-reference.mdx

```diff
@@ -373,3 +373,125 @@ Automatically opens the app in the browser on server start.
 ### `--help`
 
 Prints the help message and exits.
+
+## Advanced APIs (Experimental)
+
+If you need more control when running Astro, the `"astro"` package also exports APIs to programmatically run the CLI commands. These APIs are experimental and their API signature may change in any version.
```

Comment:
```suggestion
If you need more control when running Astro, the `"astro"` package also exports APIs to programmatically run the CLI commands. 

These APIs are experimental and their API signature may change. Any updates will be mentioned in the [Astro changelog](https://github.com/withastro/astro/blob/main/packages/astro/CHANGELOG.md) and the information below will always show the current, up-to-date information.
```
Sounds less like "you're on your own!" maybe?  :smile:

Reply:
Sounds fair 😄